scipy-1.0/paper.tex

```diff
@@ -1078,16 +1078,16 @@ \subsection*{Governance}
 The Code of Conduct specifies how breaches can be reported to a Code of Conduct Committee, and outlines procedures for the Committee's response. Our Diversity Statement ``welcomes and encourages participation by everyone''.
 
 
-\subsection*{Community beyond the SciPy library}
+\subsection*{Relation with downstream projects}
 
 The scientific Python ecosystem includes many examples
-of domain-specific software libraries building on top
-of SciPy features, and then returning to the base SciPy library
-to suggest and even implement improvements to enable
-more effective science applications. For example, there
-are common contributors to the SciPy and Astropy core
-libraries\cite{astropy-2018}, and what works well for 
-one of the codebases, infrastructures, or communities 
+of domain-specific software libraries that are built 
+on top of SciPy features. The work done on these libraries
+often turns out to be beneficial to the whole ecosystem
+being Scipy the catalyst of this cross-fertilisation process. 
```

Comment:
SciPy needs to be stylized consistently with the P capitalized.
Is "the whole ecosystem being SciPy the catalyst..." grammatically correct?

Also, I'm not sure that the rewording is 100% improved.

On one hand, it removes the phrase "to enable more effective science applications", which sounded empty to me. I don't _disagree_ with the statement at all, but think we already established that "improvements [to SciPy]" enable more effective science applications. Stating claims explicitly like this can make people question them more. There are examples of this in the news all the time!

On the other hand, the first sentence (in isolation) has less meaning now: "other libraries build on top of SciPy". That, too, has already been established in the paper. I think it's better to keep the idea that "downstream libraries contribute back to SciPy" in the same sentence (as in the original).

Reply:
> SciPy needs to be stylized consistently with the P capitalized.
Fixed

> Is "the whole ecosystem being SciPy the catalyst..." grammatically correct?
I believe so, but I am not a native speaker so I maybe wrong.

Any suggestion about how to improve the rest?

Reply:
Well, I would prefer to just remove the phrase "to enable more effective science application". I didn't have a problem with the rest of the original.

What were you trying to improve, or what did you not like about the original? Maybe we can find something that sounds better to both of us.

Reply:
I was trying to say this:

> As far as I can see, the message this paragraph is trying to convey is that the communities of downstream projects actively contribute to scipy and by doing so indirectly contibute to other projects. Thus, scipy plays a mediator and meeting point role in the python ecosystem.

that is, a little step more than just say that "downstream libraries contribute back to SciPy".

Reply:
OK, I think that's a good point to add. I'll think about it some more.